plugins/inputs/tail/tail.go

```diff
@@ -158,8 +158,9 @@ func (t *Tail) tailNewFiles(fromBeginning bool) error {
 					Pipe:      t.Pipe,
 					Logger:    tail.DiscardingLogger,
 				})
+
 			if err != nil {
-				t.acc.AddError(err)
+				t.Log.Errorf("Failed to tail file (%s): %s", filepath, err.Error())
```

Comment:
I think we want `t.Log.Debugf`, also minor style nit, for the error use "%v" and err:
```
t.Log.Debugf("Failed to tail file (%s): %v", filepath, err)
```

Reply:
I'm worried that many people won't have debug logging enabled and if there's a genuine mistake (typo?) in their config and they're running Telegraf, hiding this message behind debug would mean they lose a lot of time trying to work out what is wrong.

Reply:
This is fine then by me, as we were essentially already logging at error level, but of course you will have to set the log level for all of Telegraf to `quiet` as we don't support per module log levels yet.